CONTRIBUTING.md

```diff
@@ -0,0 +1,78 @@
+# Contributing
+Oracle welcomes contributions to this repository from anyone.
+
+If you want to submit a pull request to fix an bug or enhance an existing
+`Dockerfile`, please first open an issue and link to that issue when you
+submit your pull request.
+
+If you have any questions about a possible submission, feel free to open
+an issue too.
+
+## Pull Request Process
+1. Fork this repository
+1. Create a branch in your fork to implement the changes. We recommend using
+the issue number as part of your branch name, e.g. `1234-fixes`
+1. Ensure that any documentation is updated with the changes that are required
+by your fix.
+1. Ensure that any samples are updated if the base image has been changed.
+1. Submit the pull request. *Do not leave the pull request blank*. Explain exactly
+what your changes are meant to do and provide simple steps on how to validate
+your changes. We will assign the pull request to 2-3 people for review before
+it is merged.
+
+## Golden Rules
+
+We have some golden rules that we require all submitted `Dockerfiles` to abide
+by. These rules are provided by Oracle Global Product Security and may change
+at any time.
```

Comment:
No, these rules apply to both employees and non-employees alike. We will not accept any PRs that don't follow these rules.